command.php

```diff
@@ -139,8 +139,33 @@ private function themecheck( $theme, $path ) {
 			}
 
 			// Run Themecheck.
-			$data = tc_get_theme_data( $path . '/style.css' );
-			$success = run_themechecks( $datafiles['php'], $datafiles['css'], $datafiles['other'] );
+			if ( ! class_exists( 'WP_Theme' ) ) {
```

Comment:
@cristian-ungureanu actually there's a variable (`$theme`) instance of WP_Theme from __invoke method above.
You just need to pass it to `themecheck` method.

Reply:
Fixed it, thanks for having a look over this!